frontend/mixins/signedExtrinsic.js

```diff
@@ -0,0 +1,166 @@
+import { gql } from 'graphql-tag'
```

Comment:
Rename this file so we can collect all charts related logic here.

Reply:
done